components/pedigree/resources/src/main/resources/pedigree/controller.js

```diff
@@ -261,6 +261,55 @@ define([
                 }
             }
 
+            // Ask if the patient should be a part of a study
```

Comment:
This block should be moved into its own dialog, similar to `newPersonMenu`, mixing UI code with patient creation logic is not a very good idea